client/interface.go

```diff
@@ -0,0 +1,53 @@
+package client
+
+import (
+	"io"
+	"os"
+)
+
+// Driver handles the file system access logic.
+type Driver interface {
```

Comment:
How about using `go-storage` here directly instead of defining a new interface?

Reply:
ok, I will have a try to remove this interface